graceoncampus/goc-web-v3
# Add Thank You Pages to Rider + Driver Signups

Testing Steps
------------------------------------------------------------------------
1. yarn && yarn build && yarn start
2. navigate to localhost:3000
3. fill out driver and rider signup forms and submit
4. check to see if "thank you pages" render for 3 seconds

Resources
------------------------------------------------------------------------
- writing a good commit message: https://stackoverflow.com/questions/33097657/how-to-write-a-good-git-commit-message
- react-bootstrap: https://react-bootstrap.github.io/
